README.md

```diff
@@ -1,6 +1,64 @@
-### MIND MAPP
+# MIND MAPP
 
-Mind Mapp: An interactive mind mapping tool for programmers
+```
+.        :   ::::::.    :::.:::::::-.  
+;;,.    ;;;  ;;;`;;;;,  `;;; ;;,   `';,
+[[[[, ,[[[[, [[[  [[[[[. '[[ `[[     [[
+$$$$$$$$"$$$ $$$  $$$ "Y$c$$  $$,    $$
+888 Y88" 888o888  888    Y88  888_,o8P'
+MMM  M'  "MMMMMM  MMM     YM  MMMMP"`
+
+.        :    :::.  ::::::::::.::::::::::.
+;;,.    ;;;   ;;`;;  `;;;```.;;;`;;;```.;;;
+[[[[, ,[[[[, ,[[ '[[, `]]nnn]]'  `]]nnn]]'
+$$$$$$$$"$$$c$$$cc$$$c $$$""      $$$""    
+888 Y88" 888o888   888,888o       888o     
+MMM  M'  "MMMYMM   ""` YMMMb      YMMMb   
+
+   an interactive mind mapping tool
+```
+
+## What is this cool thing?
+
+An interactive mind map! We are using Node.js, Express, Javascript, JQuery, Pug and HTML5 Canvas to make a board that you can add ideas to, then link your ideas to other ideas. Possibly with a snazzy interface.
+
+We're testing with Mocha and Zombie and potentially Chai.
```

Comment:
Could we change this to: "We're testing with Mocha, Chai, Supertest and Zombie."